api/RTCDataChannel.json

```diff
@@ -132,7 +132,7 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": false
+              "version_added": true
```

Comment:
After looking quickly at Bugzilla, this was added when RTCDataChannel first landed in Firefox back in Firefox 18, so this should be "18".

Reply:
Thank you, you are right,  Here is release note.

[Bug 729512](https://bugzilla.mozilla.org/show_bug.cgi?id=729512)
[Datachannel implement code](https://bugzilla.mozilla.org/attachment.cgi?id=662013&action=diff#a/netwerk/sctp/datachannel/DataChannel.h_sec1)